content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -238,92 +249,62 @@ $ kubectl get no ${NODE_NAME} -o json | jq '.status.config'
 
 ```
 
-If something goes wrong, the Kubelet will report any configuration related errors
-in `Node.Status.Config.Error`. You may see one of several possible errors, which
-are detailed in a table at the end of this article. If you see any of these errors,
-you can search for the same error message in the Kubelet's log for additional details.
-
-### Edit the configuration file again
-
-To change the configuration again, we simply repeat the above workflow.
-Try editing the `kubelet_configz_${NODE_NAME}` file, changing the previously changed parameter to a
-new value.
-
-### Push the newly edited configuration to the control plane
-
-Push the new configuration to the control plane in a new ConfigMap with the
-following command:
-
-```
-$ kubectl -n kube-system create configmap my-node-config --from-file=kubelet=kubelet_configz_${NODE_NAME} --append-hash -o yaml
-```
-
-This new ConfigMap will get a new name, as we have changed the contents.
-We will refer to the new name as `NEW_CONFIG_MAP_NAME`.
-
-### Configure the Node to use the new configuration
-
-Once more, edit `Node.Spec.ConfigSource` via `kubectl edit node ${NODE_NAME}`.
-Your new `Node.Spec.ConfigSource` should look like the following,
-with `${NEW_CONFIG_MAP_NAME}` substituted as necessary:
-
-```
-configSource:
-    configMap:
-        name: ${NEW_CONFIG_MAP_NAME}
-        namespace: kube-system
-        kubeletConfigKey: kubelet
-```
+If an error occurs, the Kubelet reports it in the `Node.Status.Config.Error`
+structure. Possible errors are listed in
+[Understanding Node.Status.Config.Error messages](#understanding-node-status-config-error-messages).
+If you see an error, you can search for it in the Kubelet's log for additional
+details.
 
-### Observe that the Kubelet is using the new configuration
+#### Make more changes
 
-Once more, retrieve the Node with `kubectl get node ${NODE_NAME} -o yaml`, and
-look for a `Node.Status.Config` that reports the new configuration as `assigned`
-and `active`, with no errors.
+Follow the workflow above to make more changes and push them again. Each
+time you change the ConfigMap's contents, it gets a new name.
```

Comment:
Should we reiterate that this is due to `--append-hash`?

Maybe something like this:
"Each time you push a ConfigMap with new contents, `kubectl`'s `--append-hash` option will create the ConfigMap with a new name. As mentioned above, the safest rollout strategy is to first create a new ConfigMap, and then update the Node to use the new ConfigMap."